### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by tloc241** [sindresorhus/caprine#1905]
app took a lot of time to open, sometime even refuse to open 
if u leave it running awhile in background it just constanly force stop loop
it feel smooth to use at few first minute, after that it just delay and lagging like crazy

---

Caprine 2.56.1
Electron 20.0.3
linux 5.15.0-47-generic
Locale: vi

**Comment by dusansimic** [comment]
How is Caprine installed on your system?

**Comment by tloc241** [comment]
i'm use .deb pack

Vào Th 2, 5 thg 9, 2022 vào lúc 19:21 Dušan Simić <
***@***.***> đã viết:

**Comment by Spounka** [comment]
I second this, I have the same issue on Arch Linux.
I'm using dwm-6.2 and here's the output of my `uname -a` command:
> Linux spounka-pc 5.19.9-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 15 Sep 2022 16:08:26 +0000 x86_64 GNU/Linux

Steps to reproduce:
- Open Caprine
- Leave it open on a tag
- Continue working on other things
- Check caprine and it's totally frozen

It requires killing the process and re-lunching it for it to work again, and it goes on

**Comment by dusansimic** [comment]
@Spounka your issue sounds more like #1878

**Comment by dusansimic** [comment]
@tloc241 does it happen gradually (Caprine starts to slow down up to a point when it doesn't respond) or is it just working one moment and another it's completely frozen?

**Comment by Spounka** [comment]
> @Spounka your issue sounds more like #1878

yes it is, now reading the conversation it and thinking about it, I think it might have to do with notifications
I had the same literal issue before with discord and it showed up to be Notifications Bus being blocked or something... might give you a clue
